src/panels/lovelace/editor/hui-glance-card-editor.ts

```diff
@@ -35,37 +45,122 @@ export class HuiGlanceCardEditor extends hassLocalizeLitMixin(LitElement)
     }
 
     return html`
+      ${this.renderStyle()}
       <paper-input
         label="Title"
         value="${this._config!.title}"
         .configValue=${"title"}
+        .type=${"input"}
+        @value-changed="${this._valueChanged}"
+      ></paper-input>
+      <hui-theme-select-editor
+        .hass=${this.hass}
+        .value=${this._config!.theme}
+        .type=${"input"}
+        .configValue=${"theme"}
+        @change=${this._valueChanged}
+      ></hui-theme-select-editor><br>
+      <paper-input
+        label="Columns"
+        value="${this._config!.columns || ""}"
+        .configValue=${"columns"}
+        .type=${"input"}
         @value-changed="${this._valueChanged}"
-      ></paper-input><br>
+      ></paper-input>
+      <h4>Entities</h4>
+      <div class="entities">
+        ${this._configEntities!.map((entityConf, index) => {
+          return this.renderEntity(entityConf, index!);
+        })}
+      </div><br>
+      <paper-button
+        noink
+        raised
+        @click=${this.addEntity}
+      >Add Entity</paper-button>
+      <br><br>
       <paper-checkbox
         ?checked="${this._config!.show_name !== false}"
         .configValue=${"show_name"}
+        .type=${"checkbox"}
         @change="${this._valueChanged}"
       >Show Entity's Name?</paper-checkbox><br><br>
       <paper-checkbox
         ?checked="${this._config!.show_state !== false}"
         .configValue=${"show_state"}
+        .type=${"checkbox"}
         @change="${this._valueChanged}"
       >Show Entity's State Text?</paper-checkbox><br>
     `;
   }
 
+  private renderStyle(): TemplateResult {
+    return html`
+      <style>
+        .entities {
+          padding-left: 20px;
+        }
+      </style>
+    `;
+  }
+
+  private renderEntity(
+    entityConf: EntityConfig,
+    index: number
+  ): TemplateResult {
+    return html`
+      <ha-entity-picker
+        .hass="${this.hass}"
+        .value="${entityConf.entity || entityConf}"
+        .type=${"entity"}
```

Comment:
this is kinda weird?

Reply:
It's also bound to go wrong in case `ha-entity-picker` ever adds support for `type`.

I would suggest that you create a new component just to edit an entity row. It will handle update entity, deletion of row etc. Then you also don't have to do these hacks 😉